semanticdb/metacp/src/main/scala/scala/meta/internal/index/Index.scala

```diff
@@ -29,7 +28,7 @@ class Index {
   }
 
   def save(out: AbsolutePath): Unit = {
-    val indexAbspath = out.resolve("META-INF").resolve("semanticdb.semanticidx")
+    val indexAbspath = out.resolve("META-INF/semanticdb.semanticidx")
```

Comment:
I've always wondered if this works correctly on windows, Appveyor doesn't seem to complain but it feels wrong to hardcode `/`

Reply:
Done.